packages/dev/core/src/Gizmos/boundingBoxGizmo.ts

```diff
@@ -473,13 +473,15 @@ export class BoundingBoxGizmo extends Gizmo implements IBoundingBoxGizmo {
                             this._anchorMesh.position.copyFrom(this.attachedMesh.position);
                         }
                         // Rotate around center of bounding box
-                        this._anchorMesh.addChild(this.attachedMesh, Gizmo.PreserveScaling);

```

Comment:
Is removing PreserveScaling a breaking change? Or is it the default value?

Reply:
default value